frontend/src/Utils/api-discovery/__mocks__/staticCore.ts

```diff
@@ -0,0 +1,354 @@
+export default [
+  {
+    allResources: [
```

Comment:
@karelhala could you shed some light on which server you collected this list from? I'm assuming the main devsandbox instance - but we should clarify for the future.

Reply:
@florkbr yeah, from the devsandbox from prod environment.